SharpPcap/LibPcap/PcapDevice.cs

```diff
@@ -541,10 +541,10 @@ public static IEnumerable<RawCapture> GetSequence(ICaptureDevice dev, bool maskE
                             break;
                         packet = e.GetPacket();
                     }
-                    catch (PcapException pe)
+                    catch (PcapException)
                     {
                         if (!maskExceptions)
-                            throw pe;
+                            throw;
```

Comment:
The change is OK, but outside the scope of the PR, please move it to its own PR

Reply:
I have reverted the change here.